wallet/src/account.rs

```diff
@@ -81,6 +81,22 @@ impl Account {
         })
     }
 
+    pub fn complete_and_add_send_request<B: Backend>(
+        &mut self,
+        db_tx: &mut StoreTxRw<B>,
+        mut request: SendRequest,
+    ) -> WalletResult<Transaction> {
+        self.complete_send_request(&mut request)?;
+        self.add_transaction(db_tx, request.transaction.clone(), TxState::InMempool)?;
+        Ok(request.transaction)
+    }
+
+    fn complete_send_request(&mut self, _request: &mut SendRequest) -> WalletResult<()> {
+        // TODO implement this
+        println!("Send request ");
+        Ok(())
+    }
```

Comment:
Can we encourage more separation and move these outside this struct? The last thing we want is modules/structs that are 1000 lines long.